scheduler/src/cook/mesos/api.clj

```diff
@@ -198,6 +198,7 @@
               :status s/Str
               :submit-time PosInt
               :retries-remaining NonNegInt
+              (s/optional-key :successful) s/Bool
```

Comment:
Yea, I agree, my concern is that it would break backwards compatibility. We can deprecate the use of status or something else but I'm hesitant to change the behavior of it.